app/pages/profile/profile.html

```diff
@@ -0,0 +1,36 @@
+<ion-header>
+    <dose-amigos-toolbar [title]="title"></dose-amigos-toolbar>
+</ion-header>
+
+<ion-content padding class="profile">
+
+    <logon-required>
+
+        <ion-card>
+
+            <ion-item *ngIf="auth.user">
+                <ion-avatar item-left>
+                    <img [src]="auth.user.picture">
+                </ion-avatar>
+                <h2>
+                    {{auth.user.nickname}}
+                </h2>
+                <p>
+                    {{auth.user.email}}
+                </p>
+            </ion-item>
+
+            <ion-item>
+                <button item-right (click)="cancel()">
+                    Cancel
+                </button>
+                <button item-right (click)="logout()">
+                    Logout
+                </button>
+            </ion-item>
+
+        </ion-card>
+
+    </logon-required>
+
+</ion-content>
```

Comment:
This page shows the profile info from Google, with a log out button.
Cancel button goes back to News Feed.

<img width="376" alt="logout" src="https://cloud.githubusercontent.com/assets/6886583/16822200/d69b7f5a-4928-11e6-8227-bc4fd8302e74.png">